release/RELEASE_GUIDELINES.md

```diff
@@ -0,0 +1,31 @@
+# Release Guidelines for the Onfido JS SDK
+We follow semver for versioning. Given a version number MAJOR.MINOR.PATCH, increment the:
+
+- MAJOR version when you make incompatible API changes
+- MINOR version when you add functionality in a backwards-compatible manner
+- PATCH version when you make backwards-compatible bug fixes
+
+# Prerequisite
+In order to perform the release you need to:
+1. Download the Web SDK release configuration secret file and store it as `release/releaseSecret.js`
```

Comment:
Where from shall we download it? 
For the person that releases for the first time, it might be a bit unknown. 

If we keep these guidelines we could also include commands on how to download them if e.g. from s3?

Reply:
I wrote where from to download it but I decided to remove it because this is a public repo

Reply:
If we keep it here it makes sense but we need to share this information across the team. Personally, I don't know 🤷‍♂